docs/schema-loader.md

```diff
@@ -616,25 +616,33 @@ Auto-scaling for Cosmos DB for NoSQL is enabled only when this option is set to
 
 The following table shows the supported data types in ScalarDB and their mapping to the data types of other databases.
 
-| ScalarDB  | Cassandra | Cosmos DB for NoSQL | DynamoDB | MySQL    | PostgreSQL       | Oracle         | SQL Server      | SQLite  |
-|-----------|-----------|---------------------|----------|----------|------------------|----------------|-----------------|---------|
-| BOOLEAN   | boolean   | boolean (JSON)      | BOOL     | boolean  | boolean          | number(1)      | bit             | boolean |
-| INT       | int       | number (JSON)       | N        | int      | int              | int            | int             | int     |
-| BIGINT    | bigint    | number (JSON)       | N        | bigint   | bigint           | number(19)     | bigint          | bigint  |
-| FLOAT     | float     | number (JSON)       | N        | double   | float            | binary_float   | float(24)       | float   |
-| DOUBLE    | double    | number (JSON)       | N        | double   | double precision | binary_double  | float           | double  |
-| TEXT      | text      | string (JSON)       | S        | longtext | text             | varchar2(4000) | varchar(8000)   | text    |
-| BLOB      | blob      | string (JSON)       | B        | longblob | bytea            | RAW(2000)      | varbinary(8000) | blob    |
+| ScalarDB  | Cassandra | Cosmos DB for NoSQL | DynamoDB | MySQL    | PostgreSQL/YugabyteDB | Oracle         | SQL Server      | SQLite  |
```

Comment:
Since PostgreSQL is not the same as YugabyteDB, how about if we separate them into different columns in this table? (For example, Cassandra and SQLite have the same data types, but they are in separate columns.)

> [!NOTE]
>
> If we decide to separate PostgreSQL and YugabyteDB here, we should also separate them in other tables and tabs.

Reply:
Good question!

I don't think the data types of YugabyteDB and PostgreSQL happen to be the same. YugabyteDB has purposefully achieved the compatibility with PostgreSQL:
- https://docs.yugabyte.com/preview/explore/ysql-language-features/postgresql-compatibility/
- https://www.yugabyte.com/postgresql/postgresql-compatibility/

> Supporting [SQL](https://www.yugabyte.com/distributed-sql/sql/) (Structured Query Language), the standard language for relational databases, is also essential. This includes support for the same data types, operators, and functions used in PostgreSQL, along with similar syntax for creating and manipulating tables, indexes, and views.

I don't think they are the 100% same. But I think we can treat YugabyteDB as compatible as PostgreSQL for now to reduce our maintenance effort. We can separate them if the difference gets significant in the future. 

But I'm okay to separate them in this PR.

Reply:
Thanks for the explanation! I'm OK with keeping them together as is for now👍